core/controllers/editor_test.py

```diff
@@ -2068,8 +2039,8 @@ def setUp(self):
             'is_valid': True
         }
 
-        response = self.get_html_response('/create/%s' % self.EXP_ID)
-        self.csrf_token = self.get_csrf_token_from_response(response)
+        self.login(self.MODERATOR_EMAIL)
```

Comment:
In tests, always logout at the end (if you log in somewhere during the test).

I seem to remember @brianrodri created a nifty login-context thing recently, perhaps you could use that.

Reply:
I think it might have been [this one](https://github.com/oppia/oppia/blob/7b41c1a4e18bba956f015b772762674ce6add6b0/core/tests/test_utils.py#L1661), introduced in #6679